app/scripts/models/nodes/Node.js

```diff
@@ -38,7 +38,7 @@ define(['backbone', 'FLOOD', 'staticHelpers'], function (Backbone, FLOOD, static
                 this.set('creationName', attrs.extra.creationName);
                 this.set('displayName', attrs.extra.displayName);
             } else {
-                elems = vals.workspace.app.SearchElements.where({ creationName: attrs.typeName });
+                elems = vals.searchElements.where({ creationName: attrs.typeName });
```

Comment:
While we're at it, it looks like it should be straightforward to omit `SearchElements` from the constructor for `Node`, no?